x-pack/plugins/fleet/server/collectors/agent_collectors.ts

```diff
@@ -190,11 +212,20 @@ export const getAgentData = async (
       count: bucket.doc_count,
     }));
 
+    const upgradeDetails = ((response?.aggregations?.upgrade_details as any).buckets ?? []).map(
+      (bucket: any) => ({
```

Comment:
will it be interesting to send the `doc_count` too?

Reply:
it's not mentioned in the issue, but I was wondering that too, asked on the issue description